.planning/phases/34-nixlbench-usage-troubleshooting-and-cross-references/34-RESEARCH.md

```
@@ -0,0 +1,344 @@
+
+```bash
+nixlbench --etcd_endpoints http://etcd-server:2379 --backend UCX \
+  --initiator_seg_type VRAM --target_seg_type VRAM --scheme pairwise
+```
+
+On host 2 (target):
+
+```bash
+nixlbench --etcd_endpoints http://etcd-server:2379 --backend UCX \
+  --initiator_seg_type VRAM --target_seg_type VRAM --scheme pairwise
+```
```

markdownlint-cli2 (0.23.0)

[warning] Fenced code blocks should be surrounded by blank lines

(MD031, blanks-around-fences)